server/tserver/src/test/java/org/apache/accumulo/tserver/replication/AccumuloReplicaSystemTest.java

```diff
@@ -55,16 +55,15 @@
 import org.apache.accumulo.tserver.logger.LogEvents;
 import org.apache.accumulo.tserver.logger.LogFileKey;
 import org.apache.accumulo.tserver.logger.LogFileValue;
-import org.apache.accumulo.tserver.replication.AccumuloReplicaSystem.ReplicationStats;
-import org.apache.accumulo.tserver.replication.AccumuloReplicaSystem.WalClientExecReturn;
-import org.apache.accumulo.tserver.replication.AccumuloReplicaSystem.WalReplication;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.io.Text;
 import org.junit.Test;
 
+@SuppressWarnings("deprecation")
 public class AccumuloReplicaSystemTest {
```

Comment:
I think if you deprecate this test class, you don't need to suppress deprecations or deprecate individual methods.

Reply:
Fixed in ef1b169